extension/src/experiments/webview/messages.ts

```diff
@@ -571,4 +575,13 @@ export class WebviewMessages {
   private showPlots() {
     return commands.executeCommand(RegisteredCommands.PLOTS_SHOW, this.dvcRoot)
   }
+
+  private async copyToClipboard(text: string) {
+    await writeToClipboard(text)
```

Comment:
Should we be copying the full sha instead of the short version? When I see these context options, my first assumption is I'm copying the full sha, but on the other hand, the shortened version is probably all that's needed for most use cases.

Reply:
The short sha should work for now.